string sensor
uint8 sensor_type

